molgenis-api-tests/pom.xml

```diff
@@ -192,7 +192,7 @@
         <configuration>
           <skip>${skipITs}</skip>
           <container>
-            <containerId>tomcat8x</containerId>
+            <containerId>tomcat9x</containerId>
```

Comment:
See https://codehaus-cargo.github.io/cargo/Configuring+HTTP+2+for+Tomcat+8.5+and+above.html for how to add http/2